ports/glad/vcpkg.json

```
@@ -0,0 +1,125 @@
+{
+  "name": "glad",
+  "version-string": "0.1.34",
```

```suggestion
  "version": "0.1.34",
```
Using a more intelligent version scheme enables our new versioning feature to work better